WidgetExtension/Up Next/UpNextStyles.swift

```diff
@@ -4,6 +4,128 @@ import SwiftUI
 let darkBackgroundColor = Color(UIColor.tertiarySystemFill)
 let lightBackgroundColor = Color(.clear)
 
+struct PCWidgetColorScheme {
+    let topBackgroundColor: Color
+    let bottomBackgroundColor: Color
+    let topButtonBackgroundColor: Color
+    let bottomButtonBackgroundColor: Color
+    let topTextColor: Color
+    let bottomTextColor: Color
+    let topButtonTextColor: Color // foreground color?
+    let bottomButtonTextColor: Color // foreground color?
+    let iconAssetName: String
+    let filterViewBackgroundColor: Color
+    let filterViewTextColor: Color
+    let filterViewIconAssetName: String
+}
+
+let rgbMax = 255.0
```

Comment:
I wonder if this could be added to an extension initializer on `Color`? It would make this a little more self contained and avoid polluting the `WidgetExtension` with this value.

```swift
extension Color {
    /// A convenience initializer that takes RGB values 0-255 and converts to the 0-1 range expected by Color.
    /// - Parameters:
    ///   - red: A value 0-255 for the red component
    ///   - green: A value 0-255 for the green component
    ///   - blue: A value 0-255 for the blue component
    init(red: Int, green: Int, blue: Int) {
        let rgbMax = 255
        Color(.sRGB, red: red/rgbMax, green: green/rgbMax, blue: blue/rgbMax)
    }
}
```